drivers/adc/adc_esp32.c

```diff
@@ -57,6 +57,10 @@ LOG_MODULE_REGISTER(adc_esp32, CONFIG_ADC_LOG_LEVEL);
 
 #define ADC_DMA_BUFFER_SIZE	DMA_DESCRIPTOR_BUFFER_MAX_SIZE_4B_ALIGNED
 
+#if CONFIG_SOC_SERIES_ESP32S2 || CONFIG_SOC_SERIES_ESP32C3
+#define ESP32_ADC2_UNIT 1
```

Comment:
Where is it used?

Reply:
It's used on line 662 of this same file.

Reply:
Sound something isn't correct.
We do have  `ADC_UNIT_1` and `ADC_UNIT_2` available in the code. Why having `ESP32_ADC2_UNIT` at all?
The only instance where it is used is at
```c
#if CONFIG_SOC_SERIES_ESP32S2 || CONFIG_SOC_SERIES_ESP32C3
	if (conf->unit == ESP32_ADC2_UNIT) {
		adc2_init_code_calibration();
	}
#endif /* CONFIG_SOC_SERIES_ESP32S2 || CONFIG_SOC_SERIES_ESP32C3 */
```
It seems you can just trash `ESP32_ADC2_UNIT` and use `ADC_UNIT_1`?